easybuild/toolchains/mpi/intelmpi.py

```diff
@@ -67,6 +71,23 @@ def _set_mpi_compiler_variables(self):
 
         super(IntelMPI, self)._set_mpi_compiler_variables()
 
+    def _set_mpi_variables(self):
+        """Set the other MPI variables"""
+
+        if (LooseVersion(self.version) >= LooseVersion('2019')):
+            lib_dir = [os.path.join('intel64', 'lib', 'release')]
+            incl_dir = [os.path.join('intel64', 'include')]
+
+            for root in self.get_software_root(self.MPI_MODULE_NAME):
+                self.variables.append_exists('MPI_LIB_STATIC', root, lib_dir,
+                                             filename="lib%s.a" % self.MPI_LIBRARY_NAME)
+                self.variables.append_exists('MPI_LIB_SHARED', root, lib_dir,
+                                             filename="lib%s.so" % self.MPI_LIBRARY_NAME)
+                self.variables.append_exists('MPI_LIB_DIR', root, lib_dir)
+                self.variables.append_exists('MPI_INC_DIR', root, incl_dir)
+
+        super(IntelMPI, self)._set_mpi_variables()
```

Comment:
@akesandgren Shouldn't the `super` call be done first, to avoid that `Mpi._set_mpi_variables` overrides what is being set above?

Reply:
It won't overwrite anything because the super call won't resolve to a directory. For >= 2019 the defaults doesn't exist.

A more important thing is that MPI_LIB_DIR and MPI_INC_DIR must be single-dir values, not PATH-type. That's how they are used.

Reply:
i.e., that code does :-separator appending, not overwrite.